tasks/tasks.go

```diff
@@ -38,12 +40,37 @@ func LoadStorageEngine(ctx context.Context, config *tsbridge.Config) (storage.Ma
 	}
 }
 
-// Sync updates all configured metrics.
-func Sync(ctx context.Context, config *tsbridge.Config, metrics *tsbridge.Metrics, store storage.Manager) error {
-	metricCfg, err := tsbridge.NewMetricConfig(ctx, config, store)
+// Checks if the metric config file has been updated, and updates the cached file stats if it has.
+func metricConfigFileUpdated(ctx context.Context, config *tsbridge.Config, metricCfgFs *os.FileInfo) (bool, error) {
+	currMetricCfgFs, err := os.Stat(config.Options.Filename)
```

Comment:
Readbility nit:
In Go, variable names should be short, proportional to the size of its scope.
examples:
 'f' is preferred to 'file'
'c' is preferred to 'count'
'count' is preferred to 'countUsers'
... and so on

The func here already declares that it is dealing with the config, so the name doesn't need to repeat that info.

`f, err := os.Stat(....)`

Reply:
thanks for the tip! I've updated it to be `fs` for file stats.